src/site/content/en/metrics/lcp/index.md

```diff
@@ -101,10 +101,8 @@ considered for Largest Contentful Paint are:
   [CSS gradient](https://developer.mozilla.org/docs/Web/CSS/CSS_Images/Using_CSS_gradients))
 - [Block-level](https://developer.mozilla.org/docs/Web/HTML/Block-level_elements)
   elements containing text nodes or other inline-level text elements children.
-
-Note, restricting the elements to this limited set was intentional in order to
-keep things simple in the beginning. Additional elements (like the full `<svg>` support or
-`<video>` elements without poster images) may be added in the future as more research is conducted.
+- The first frame painted for `<video>` elements, negating the need for the `poster` attribute to used for autoplaying video as of [August 2023](https://chromium.googlesource.com/chromium/src/+/refs/heads/main/docs/speed/metrics_changelog/lcp.md)
+- The first frame of an animated image format, such as animated GIFs as of [August 2023](https://chromium.googlesource.com/chromium/src/+/refs/heads/main/docs/speed/metrics_changelog/lcp.md)
```

Comment:
Rephrasing and removing the implication about not needing `poster`. LMK if you do think we should discourage `poster`.

```suggestion
- The first frame painted for autoplaying `<video>` elements (as of [August 2023](https://chromium.googlesource.com/chromium/src/+/refs/heads/main/docs/speed/metrics_changelog/lcp.md))
- The first frame of an animated image format, such as animated GIFs (as of [August 2023](https://chromium.googlesource.com/chromium/src/+/refs/heads/main/docs/speed/metrics_changelog/lcp.md))
```